bank-sdk/sdk/src/test/java/net/gini/android/bank/sdk/capture/skonto/usecase/GetSkontoRemainingDaysUseCaseTest.kt

```diff
@@ -0,0 +1,17 @@
+package net.gini.android.bank.sdk.capture.skonto.usecase
+
+import org.junit.Test
+import java.time.LocalDate
+
+
+class GetSkontoRemainingDaysUseCaseTest {
+
+    @Test
+    fun `Skonto remaining days should be calculated correctly`() {
```

Comment:
[Suggestion] What do you think about adding other tests for this method? you can verify today, and past as well.